apps/ledger-live-mobile/src/newArch/features/Market/screens/MarketDetail/components/MarketGraph/index.tsx

```diff
@@ -1,43 +1,34 @@
 import React, { useMemo, useCallback, memo } from "react";
 import { useTheme } from "styled-components/native";
 import { Flex, GraphTabs, InfiniteLoader, Transitions } from "@ledgerhq/native-ui";
-import { rangeDataTable } from "@ledgerhq/live-common/market/utils/rangeDataTable";
 import { useTranslation } from "react-i18next";
-import { SingleCoinProviderData } from "@ledgerhq/live-common/market/MarketDataProvider";
 import Graph from "~/components/Graph";
 import getWindowDimensions from "~/logic/getWindowDimensions";
 import { Item } from "~/components/Graph/types";
+import { RANGES } from "~/newArch/features/Market/utils";
 
 const { width } = getWindowDimensions();
 
 function MarketGraph({
   setHoverItem,
-  chartRequestParams,
-  loading,
-  loadingChart,
+  isLoading,
   refreshChart,
   chartData,
+  range,
 }: {
   setHoverItem: (_: Item | null | undefined) => void;
-  chartRequestParams: SingleCoinProviderData["chartRequestParams"];
-  loading?: boolean;
-  loadingChart?: boolean;
+  isLoading?: boolean;
   refreshChart: (_: { range: string }) => void;
-  chartData: Record<string, [number, number][]>;
+  chartData?: Record<string, [number, number][]>;
```

Comment:
Maybe create a general type as you use it somewhere else. 
And maybe make [number,number] more explicit with a type chartPoint